llvm/test/Transforms/IndVarSimplify/preserving-debugloc-rem-div.ll

```diff
@@ -0,0 +1,255 @@
+; Test that the debug information is propagated correctly to the new instructions
+; RUN: opt < %s -passes=indvars -S | FileCheck %s
+
+define void @test_srem_urem(ptr %a) !dbg !5 {
+; CHECK-LABEL: define void @test_srem_urem(
+; CHECK:    [[REM_UREM:%.*]] = urem i32 [[I_01:%.*]], 2, !dbg [[DBG20:![0-9]+]]
+; CHECK:      #dbg_value(i32 [[REM_UREM]], [[META11:![0-9]+]], !DIExpression(), [[DBG20]])
+;
+entry:
+  br label %for.body, !dbg !18
+
+for.body:                                         ; preds = %for.body, %entry
+  %i.01 = phi i32 [ 0, %entry ], [ %inc, %for.body ], !dbg !19
+  tail call void @llvm.dbg.value(metadata i32 %i.01, metadata !9, metadata !DIExpression()), !dbg !19
```

Comment:
Thanks I've updated the test using `--debugify-level=locations`. Since the pass test induction variables which will need a loop, I dont think we can reduce the tests further. I have taken them from existing tests in the same folder. I had already reduced test_rem_num_zero as much as possible.

Reply:
If these changes look okay and you have no further comments, could you please go ahead and merge this? I do not have commit access. Thanks.